packages/common/src/api/purchases.ts

```diff
@@ -0,0 +1,165 @@
+import { full } from '@audius/sdk'
```

Comment:
This file got moved from `users` because I had intended to wire up pre-fetching of users as part of fetching purchases/sales. But it turns out we don't yet have a bulk users endpoint. However, leaving this here in anticipation of adding one in the near future.